ioos_catalog/models/dataset.py

```diff
@@ -39,6 +40,8 @@ class Dataset(BaseDocument):
                 'keywords'          : [unicode],  # Search keywords
                 'variables'         : [unicode],  # Environmental properties measured by this dataset
                 'asset_type'        : unicode,    # See the IOOS vocablary for assets: http://mmisw.org/orr/#http://mmisw.org/ont/ioos/platform
+                'time_min': datetime, # start time of this service
+                'time_max': datetime, # end time of this service
```

Comment:
In the future, I like to keep the tabular style, it makes it easier to read, I think.